### DISCUSSION TITLE
Pre-heading

### DISCUSSION THREAD

**Discussion by Rozerinay** [nl-design-system/backlog#286] [Component suggestions]
<img width="480" alt="visuele weergave van de pre-heading component" src="https://github.com/user-attachments/assets/7fcd0ec8-f596-4a00-b631-9b57797097be">

We willen graag meer leren over de **Pre-heading** component. In welke vorm, of vormen, wordt deze momenteel ingezet en toegepast binnen jouw organisatie?

Deel hieronder een link naar Figma, Storybook, Github of screenshots. Met deze informatie kunnen we bewijzen dat de component onderdeel moet worden van de NL Design System component library.

Daarnaast wordt het delen van onderzoek gewaardeerd. Dus heb je zelf onderzoek gedaan en zijn er inzichten voor dit component? Dan horen we dit ook graag!

[Link naar bericht in Slack](https://codefornl.slack.com/archives/C7KDQRXMZ/p1728646202902079)

---

## Naam
Pre-heading

## Doel
Een ondersteunende tekst die boven een kop staat.

## Gerelateerde componenten
[Rich Text Content](https://github.com/orgs/nl-design-system/discussions/289), [Subheading](https://github.com/orgs/nl-design-system/discussions/292), [Heading Group](https://github.com/orgs/nl-design-system/discussions/323)

## Onderzoek
[Contactformulier voor WMEBV - Gebruikersonderzoeken.nl](https://arc.net/l/quote/ioefituz)
Gebruikersonderzoek waarin onder andere de stapaanduiding in een formulier werd getest. Deze stond boven de Heading-2. Uit dit onderzoek bleek dat participanten, die niet bewust op zoek gingen naar de stappen, deze uiteindelijk wel vonden. Ze gaven echter aan dat het prettiger zou zijn als de stapaanduiding groter en duidelijker op de pagina stond. 

[Heading Group - Storybook gemeente Utrecht](https://nl-design-system.github.io/utrecht/storybook/?path=/docs/css_css-heading-group--docs)
In de docs staat aangegeven dat om aan [WCAG eis 1.3.2](https://www.w3.org/TR/WCAG21/#meaningful-sequence) te voldoen, je de HTML van Heading plaatst vóór de HTML van de Pre-heading. De volgorde kan daarna met CSS aangepast worden.

**Comment by Rozerinay** [comment]
## RVO

- [Storybook](https://nl-design-system.github.io/rvo/?path=/story/pagina-s-voorbeelden-cmor-zaak--default)

---

### Screenshots / Varianten

<img width="513" alt="image" src="https://github.com/user-attachments/assets/96fba2ae-4109-4611-9426-4cd96c1323a6"><br>

<img width="285" alt="image" src="https://github.com/user-attachments/assets/75553826-beed-4c30-abb0-eb44ebdd0c64">


---

### Notities
- Geen component gevonden.
- In de voorbeelden hierboven is `rvo-text--subtle` gebruikt boven een Heading 2.
- Bestaat uit 1 woord.

**Comment by Rozerinay** [comment]
## Utrecht

- [Storybook](https://nl-design-system.github.io/utrecht/storybook/?path=/docs/css_css-pre-heading--docs)
- [Github](https://github.com/nl-design-system/utrecht/tree/main/components/pre-heading)

---

### Screenshots / Varianten

**Storybook**

<img width="440" alt="image" src="https://github.com/user-attachments/assets/3bd3c920-c287-46c4-85a7-96a80a7ae02f">

---

### Notities
Wordt gemarkeerd met het HTML-element `<p>`, zodat het binnen een `<hgroup>` kan worden gebruikt.

**Comment by Rozerinay** [comment]
## Frameless

[Publieke website](https://arc.net/l/quote/nobfeeem)

---

### Screenshots / Varianten

Het gaat hier om de regel 'Succescriterium 1.1.1 Niveau A'
<img width="586" alt="image" src="https://github.com/user-attachments/assets/67740d35-b08c-43c6-9756-5ce7579add25">

---

### Notities
- Pre-heading bestaat uit de componenten Link en Data Badge. 
- Staat boven een Heading 3.
- Er wordt gebruik gemaakt van `utrecht-pre-heading`.

**Comment by rianrietveld** [comment]
Twee opmerkingen voor de waarborging van de toegankelijkheid.

1: Alle info die bij een kopje hoort, moet in de code onder het kopje staan.
Anders missen screenreadergebruikers, die rechtstreeks naar een kopje springen, de info die erboven staat.
Dit is op te lossen door in de code de pre-heading onder het kopje te zetten en deze visueel, met CSS naar boven te verplaatsen.
Lees het WCAG-succescriterium: [1.3.2 Betekenisvolle volgorde](https://nldesignsystem.nl/wcag/1.3.2).

2: Gebruik liever geen ALL CAPS voor de pre-heading.
Dit is lastiger te lezen omdat het woordbeeld verandert in een rechthoek. Dit is speciaal lastig voor mensen met dyslexie.
Dit is geen WCAG-succescriterium maar wel een [NL Design System Stijl richtlijn](https://nldesignsystem.nl/richtlijnen/stijl/typografie).

**Comment by Rozerinay** [comment]
## Den Haag

[Publieke website](https://www.denhaag.nl/nl/nieuws/doe-mee-aan-de-haagse-mantelzorg-driedaagse/)

---

### Screenshots / Varianten

<img width="695" alt="image" src="https://github.com/user-attachments/assets/edc052d4-bf70-46cb-8274-07db649fe6a4">

---

### Notities
- Geen component gevonden. 
- Heeft de class `denhaag-post-info`. 
- Wordt gebruikt in nieuwsartikelen om de publicatiedatum aan te geven. 
- Staat boven een Heading-1.

**Comment by Rozerinay** [comment]
## Rotterdam

---

### Screenshots / Varianten

<img width="411" alt="image" src="https://github.com/user-attachments/assets/bd457e43-8a52-4024-99c4-94c6de62b8cc">

Pagina: [Regelhulp Geldzorgen](https://www.rotterdam.nl/regelhulp-geldzorgen?step=1&choices=)

<img width="579" alt="image" src="https://github.com/user-attachments/assets/021b7476-918a-4d94-bbd0-007713dbadc8">

Pagina: [Bestrijding mediterraan draaigatje in Ommoord](https://www.rotterdam.nl/bestrijding-mediterraan-draaigatje-in-ommoord)

---

### Notities

- Geen component gevonden.
- Wordt gebruikt om het aantal stappen en de huidige stap weer te geven.
- Wordt ook gebruikt bij nieuwsartikelen om de datum, het onderwerp en de leestijd weer te geven.
- Staat boven een Heading-1 en Heading-3.

**Comment by Rozerinay** [comment]
## KVK

---

### Screenshots / Varianten

<img width="533" alt="image" src="https://github.com/user-attachments/assets/27844176-9cb3-4e6d-8dc2-578adb76571b">

Pagina: [Moet je organisatie UBO-opgave doen?](https://www.kvk.nl/ubo/moet-je-organisatie-ubo-opgave-doen/)

<img width="500" alt="image" src="https://github.com/user-attachments/assets/3e601b60-ebfa-4f8d-ad56-b41f362a5e5f">

Pagina: [Wie zijn de UBO's van je organisatie?](https://www.kvk.nl/ubo/wie-zijn-de-ubos-van-je-organisatie-g2/)

---

### Notities

- Geen component gevonden.
- Wordt 'Chapeau' genoemd. 
- Wordt gebruikt om het aantal stappen en de huidige stap weer te geven.
- Staat boven een Heading-1.

**Comment by AlineNap** [comment]
## Logius | Product: Mijn Aansluitingen

- [Designs](https://www.figma.com/design/aJzoatSaBDF8lNqqCLa6no/MijnLogius?node-id=292-16568&t=O6PCKkZ9NWwnQOeX-1)
- [Component a la NLDS](https://www.figma.com/design/jzxqJv4PtgHJdmOJwY6lYg/NLDS---Bibliotheek---LUX-tokens?node-id=15707-80970)

<img width="450" alt="Screenshot 2024-10-16 at 10 56 56" src="https://github.com/user-attachments/assets/6b929618-0675-4ecc-9abe-5ab2c050920a">

---
### Notities
- Wordt gebruikt om het aantal stappen en de huidige stap weer te geven.
- Staat boven een H2
- Gebruikt Utrecht component op aanraden van NLDS

**Comment by AlineNap** [comment]
Zelfde [opmerking als bij de subheading](https://github.com/orgs/nl-design-system/discussions/292#discussioncomment-10957239): 

Ik vraag mij af of de subheading en pre heading een apart component zijn. Hergebruik je deze onderdelen en haar stijling op andere plekken dan de heading-group? Zo niet zou ik deze componenten onderdeel maken van de heading-group a la:

prefix.heading-group.preheading.*
prefix.heading-group.subheading.*

Net als een label in een button of accordion.
Of de content en attribution in de blockqoute

**Comment by Rozerinay** [comment]
Rian's aanvulling in het [draadje van het hiep hoi bericht](https://codefornl.slack.com/archives/C7KDQRXMZ/p1729774156687989?thread_ts=1729767600.833249&cid=C7KDQRXMZ).

> Ja, mits [WCAG 1.3.2 Betekenisvolle volgorde](https://nldesignsystem.nl/wcag/1.3.2) gerespecteerd wordt.

**Comment by Rozerinay** [comment]
We sluiten de Discussion voor deze component, omdat deze niet langer een zelfstandige component is maar onderdeel uitmaakt van de [Heading Group](https://github.com/orgs/nl-design-system/discussions/323).
